deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml

```
@@ -86,7 +96,115 @@ spec:
       kind: HiveTable
       name: hivetables.metering.openshift.io
       version: v1
-  description: An Ansible-based OpenShift Operator based off of Operator-SDK to create and upload Openshift cost-usage reports to Cost Management
+  description: |-
+
+    ## Introduction
+
+    The `cost-mgmt-operator` is an OpenShift Operator used to obtain OpenShift usage data and upload it to [cost managment](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/getting_started_with_cost_management/index). The Operator utilizes [Ansible](https://www.ansible.com/) and [operator-metering](https://github.com/operator-framework/operator-metering) to create, collect, package, and upload cost and usage reports to cost management. For more information, reach out to cost-mgmt@redhat.com.
+
+    ## Features and Capabilities
+
+    The `cost-mgmt-operator` does two things:
+    1. Utilizes openshift-metering to create cost and usage reports specific to cost management.
+    2. Collects and packages these reports inside of a tarball which gets uploaded to cost management through [cloud.redhat.com](https://cloud.redhat.com/).
+
+    ## Configuring the cost management operator
+
+    ### Configuring authentication
+    An authentication secret must exist inside of the `openshift-metering` project for uploading to [cloud.redhat.com](https://cloud.redhat.com/).
+    1. Copy the following into a file called `auth_secret.yaml`:
+
+        ```
+        ---
+
+        kind: Secret
+        apiVersion: v1
+        metadata:
+          name: auth-secret-name
+          namespace: openshift-metering
+          annotations:
+            kubernetes.io/service-account.name: cost-mgmt-operator
+        data:
+          username: >-
+            Zm9v
+          password: >-
+            Zm9v
+          token: >-
+            Zm9v
```

```suggestion
            Y2xvdWQucmVkaGF0LmNvbSB0b2tlbg==
```
Are we basically requiring that the user pull the JSON value from the pull secret .dockerconfigjson auths.cloud.openshift.com.auth?

I'm not sure the doc below covers that. 

I think we'd want something like:
```
oc get secret pull-secret -o "jsonpath={.data.\.dockerconfigjson}" | base64 --decode | jq '.auths."cloud.openshift.com".auth'
```

---

Great point. I will add this!!